admin/class-h5p-content-admin.php

```diff
@@ -344,7 +345,30 @@ public function process_new_content($echo_on_success) {
       }
     }
 
-    if ($this->content !== NULL) {
+    // Check for lifelabs column map submission
+    if ($this->content !== null && filter_input(INPUT_POST, 'lifelabs-column-map-only', FILTER_VALIDATE_BOOLEAN) === true) {
+      // Get form data
+      $columnValueRemaps = filter_input(INPUT_POST, 'h5p-column-name-remap');
+      $rowValueRemaps = filter_input(INPUT_POST, 'h5p-row-value-remap');
+      //Set maps in DB
+      $update_columns_data = $wpdb->query($wpdb->prepare(
```

Comment:
Can you explain what this is actually doing? What is a map? What does this query actually change?